src/System.Data.SqlClient/tests/FunctionalTests/DiagnosticTest.cs

```diff
@@ -27,6 +27,7 @@ public class DiagnosticTest : RemoteExecutorTestBase
 
         [Fact]
         [ActiveIssue("dotnet/corefx #17925", TargetFrameworkMonikers.NetFramework)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot)] // Internals reflection not supported on uapaot
```

Comment:
Reflection is blocked when calling GetRuntimeProperty(), which is how all the tests retrieve the data they validate.